phpBB/phpbb/db/tools.php

```diff
@@ -1574,7 +1574,15 @@ function sql_prepare_column_data($table_name, $column_name, $column_data)
 				}
 				else
 				{
-					$default_val = "'" . $column_data[1] . "'";
+					// Integers need to have 0 instead of empty string as default
+					if (preg_match('/int/i', $column_type))
```

Comment:
I thought we might also support TINT but as it turns out there is no such thing in postgresql. Right now this is case insensitive though. Changing it to strpos() will modify this. Again, this should be fine as we always use uppercase for the column type in postgres.
I'll change this when I add the tests later today.